.github/workflows/update-v8.yml

```diff
@@ -0,0 +1,66 @@
+name: V8 update
+on:
+  schedule:
+    # Run once a week at 00:05 AM UTC on Sunday.
+    - cron: 5 0 * * 0
+
+  workflow_dispatch:
+    inputs:
+      id:
+        description: The ID of the job to run
+        required: true
+        default: all
+        type: choice
+        options:
+          - all
+          - minor
+          # preparing for v8 major
```

Comment:
TBH I don't think automating major V8 updates is possible.

Reply:
`git node v8 major` is only the first step. There are several other things to do after it and while some of them might also be automated, I always have to do specific changes that are different for each update.

Reply:
ok then I guess it makes sense to put everything in `tools.yml` PTAL